src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * as Client from './launcher';
+export { default as offline } from './authenticator';
```

Comment:
If you need to rename the default, then why not just use `export` instead if `export default`